src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier{
+    public int getRandomBall(){
+        Random random = new Random();
+        return random.nextInt(101) ;
```

Comment:
The magic number 101 should be extracted as a named constant to give it meaning and to avoid magic numbers in the code.